### ISSUE TITLE
Download progress not announced to user

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#6409]
Currently the progress text e.g. which response is being downloaded is not announced to the user. Announcing each download would be very verbose for an AT user. This probably needs some thought, something along the lines of start, progress and finish.

<img width="1151" height="678" alt="Image" src="https://github.com/user-attachments/assets/96581e45-0458-44a8-a7bb-5f57814b7ceb" />
